### PR TITLE
MARP-957: Add URLs for LinkedIn, Xing, Youtube, Facebook

### PR DESCRIPTION
**Pull request by vhhoang-axonivy**

### CODE REVIEW

File: marketplace-ui/src/app/shared/components/footer/footer.component.spec.ts [hunk 1]
```diff
@@ -70,4 +71,19 @@ describe('FooterComponent', () => {
     expect(getComputedStyle(logo.nativeElement).textAlign).toBe('center');
     expect(getComputedStyle(ivyPolicy.nativeElement).textAlign).toBe('center');
   });
+
+  it('should navigate to the correct URL when the link is clicked', () => {
```
Comment (phhung-axonivy) [inline marketplace-ui/src/app/shared/components/footer/footer.component.spec.ts:75]: `when the link is clicked` -> `when the link icon is clicked'`

**Review by phhung-axonivy — CHANGES_REQUESTED** [review]

File: marketplace-ui/src/app/shared/components/footer/footer.component.spec.ts (hunk 1 above)
Comment (vhhoang-axonivy) [reply marketplace-ui/src/app/shared/components/footer/footer.component.spec.ts]: Adjusted test according to feedback

File: marketplace-ui/src/app/shared/constants/common.constant.ts [hunk 2]
```diff
@@ -36,21 +36,25 @@ export const NAV_ITEMS: NavItem[] = [
 
 export const SOCIAL_MEDIA_LINK = [
   {
-    styleClass: 'fab fa-linkedin',
-    url: '/'
+    styleClass: 'fa-brands fa-linkedin-in',
```
Comment (phhung-axonivy) [inline marketplace-ui/src/app/shared/constants/common.constant.ts:39]: add class `text-primary` to change the text color to `#1B1B1B`
image
-> `fa-brands fa-linkedin-in text-primary`
Adapt for other link icons

Comment (vhhoang-axonivy) [reply marketplace-ui/src/app/shared/constants/common.constant.ts:39]: Added text-primary for correct footer icon color

**Review by phhung-axonivy — APPROVED** [review]
LGTM

**Pull request merged**